fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/connectors/ComarchApiProductConnector.java

```diff
@@ -1,4 +1,44 @@
 package pl.fakturogen.comarch.connector.connectors;
 
+import okhttp3.Response;
+import org.springframework.stereotype.Component;
+import pl.fakturogen.comarch.connector.converters.ComarchProductConverter;
+import pl.fakturogen.comarch.connector.dto.ComarchProductDTO;
+import pl.fakturogen.comarch.connector.mapper.ComarchProductMapper;
+import pl.fakturogen.comarch.connector.model.ComarchProduct;
+
+import java.io.IOException;
+import java.util.List;
+
+@Component
 public class ComarchApiProductConnector {
+
+    private HttpConnectorUtils httpConnectorUtils;
+    private ComarchProductConverter comarchProductConverter;
+    private ComarchProductMapper comarchProductMapper;
+
+    private final String getAllEndpoint = "https://app.erpxt.pl/api2/public/products";
+    private final String getByIdEndpoint = "https://app.erpxt.pl/api2/public/products/";
+
```

Comment:
In HttpConnectorUtils method httpGetById has modifier for url (url + "/" + id). It is not necessary to set two endpoints (without and with "/").